### PR TITLE
env var mapboxApiAccessToken

### PR DESCRIPTION
**Pull request by diw2**
add MapboxAccessToken env var to portal from cloud-automation

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ const plugins = [
   new webpack.EnvironmentPlugin(['WORKSPACE_URL']),
   new webpack.EnvironmentPlugin(['WTS_URL']),
   new webpack.EnvironmentPlugin(['MANIFEST_SERVICE_URL']),
+  new webpack.EnvironmentPlugin(['MapboxAccessToken']),
```
Comment (mfshao) [inline webpack.config.js:39]: can we change the name of this env var to sth like `MAPBOX_ACCESS_TOKEN` so all env vars have similar format?

Comment (diw2) [reply webpack.config.js]: https://github.com/uc-cdis/cloud-automation/pull/1200/files FYI

File: runWebpack.sh [hunk 2]
```diff
@@ -160,6 +160,7 @@ if [[ "$NODE_ENV" == "dev" || "$NODE_ENV" == "auto" ]]; then
   echo ./node_modules/.bin/webpack-dev-server
   ./node_modules/.bin/webpack-dev-server
 else
+  export NODE_OPTIONS='--max-old-space-size=4096'
```
Comment (paulineribeyre) [inline runWebpack.sh:163]: what is this?

Comment (mfshao) [reply runWebpack.sh:163]: this is to allow node to allowcate more mem (default is 512MB i think). 

we have this in the Dockerfile but don't know why not here. And portal is so fat that it need this to spin up...

Comment (paulineribeyre) [reply runWebpack.sh:163]: ok, is this related to covid19? if not, can we make a separate PR to merge this to master? this PR will only be merged to branch feat/covid19dashboard for now

Comment (mfshao) [reply runWebpack.sh:163]: yeah portal on this branch won't start up if don't have this 😞 

i guess by adding more into portal eventually the master branch will gonna need this one day

Comment (paulineribeyre) [reply runWebpack.sh:163]: just adding 1 environment variable triggered this?? 😱

Comment (mfshao) [reply runWebpack.sh:163]: it sounds unrealistic but 😕

**Comment by paulineribeyre** [comment]
> could we add a warning at startup when MAPBOX_API_TOKEN is not set?

sorry, changed my mind 🤦‍♀️it shouldn't be at startup because commons that don't use map should not get a warning all the time haha. We can put the warning in the covid19dashboard/index.jsx file instead

**Pull request merged**